test/lib/bamboo/local_adapter_test.exs

```diff
@@ -19,15 +19,14 @@ defmodule Bamboo.LocalAdapterTest do
     assert SentEmail.all == [email]
   end
 
-  test "handle_config makes sure that the DeliverImmediatelyStrategy is used" do
+  test "use DeliverImmediatelyStrategy if strategy is unspecified" do
```

Comment:
And then we can remove this test completely since `handle_config` would just return the config as-is